test/test-csharp.bat

```diff
@@ -0,0 +1,20 @@
+@echo off
```

Comment:
All of these languages specific runners could just call into a single script.

Reply:
yes, absolutely 👍